app/Repositories/BaseRepository.php

```diff
@@ -0,0 +1,326 @@
+<?php
+namespace App\Repositories;
+
+use Illuminate\Support\Facades\DB;
+use App\Repositories\RepositoryInterface;
+
+abstract class BaseRepository implements RepositoryInterface
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Repositories/BaseRepository.php</br>**Line**: 140</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line